desc/objectives/objective_funs.py

```diff
@@ -1106,42 +1137,67 @@ def print_value(self, *args, **kwargs):
             fmin = jnp.min(f)
             fmean = jnp.mean(f * w) / jnp.mean(w)
 
+            f0 = jnp.abs(f0) if abserr else f0 if val0 is not None else f
+            f0max = jnp.max(f0) if val0 is not None else fmax
+            f0min = jnp.min(f0) if val0 is not None else fmin
+            f0mean = jnp.mean(f0 * w) / jnp.mean(w) if val0 is not None else fmean
+
             print(
                 "Maximum "
                 + ("absolute " if abserr else "")
-                + self._print_value_fmt.format(fmax)
+                + print_value_fmt.format(f0max, fmax)
```

Comment:
If you provide more values to .format then there are spaces in the string for such values, it is fine with that? No errors/warnings are thrown?

when `val0` is None there is only a single slot for a value in the string

Reply:
hm seems so, ok nvm, maybe though a comment here saying "only first value will be printed which is fmin when `val0` is None" would make it clearer